docs/component-decorators.md

```diff
@@ -233,3 +233,26 @@ class extends Component {
 ```
 
 Computed properties can freely be used inside `TemplateComponent's` `template` method and will trigger rerender.
+
+## @module
+
+This decorator is a simple tool for retrieving any registered module instance. It uses `$app.getModule` under the hood. Similarly to `@el`, it should be used on class field. It accepts either module's name, or module itself. Second way is suggested, as it's independent of possibility of changing module's name.
+
+Example:
+
+```ts
+import { module, Component, register } from 'ovee.js'
+
+@register('my-component')
+class MyComponent extends Component {
+    @module('MyModule')
+    myModule;
+
+    @module(OtherModule)
+    otherModule;
+
+    // in TypeScript with typing
+    @module(OtherModule)
+    otherModule: OtherModule;
```

Comment:
If we type the property, maybe we could get the expected module type from the field's typing itself with reflection (`Reflect.getMetadata("design:type", target, key)`)?

This would allow us for syntax like that:

```
@module()
otherModule: OtherModule;
```

This should make the syntax compact and less verbose.